app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -2,6 +2,7 @@
 
 import android.Manifest;
 import android.annotation.TargetApi;
```

Comment:
I'm not going to force you on this one, but I feel like this commit can still be split up into meaningful smaller commits.

Your commit text you have is kind of an evidence for that. Those 4 things you list aren't tightly related at all. You could easily split this up into 4 different commits.

Here's some explanation to encourage to split this up into commits. There are a couple of reasons for asking to do this:
1. Much easier to review.
2. Easier to drop a commit and push the rest if one of the changes has a bug that's not easy to fix.
3. More importantly, if we find any bugs in the future in one of your split up commits, we can easily revert that one commit instead of reverting one large commit and losing all the features/benefits because one of them was broken.
4. Reverting smaller commits is also a lot less likely to conflict in the future than one large commit.